guardrails/utils/sql_utils.py

```diff
@@ -2,7 +2,9 @@
 from pathlib import Path
 from typing import List, Optional
 
+import sqlalchemy
 from pydantic import ValidationError
+from sqlalchemy import text
```

Comment:
nit: When you're adding docstrings, can you import guard this? E.g. https://github.com/ShreyaR/guardrails/blob/b7e2a4d1104772358bf298f38bf07491d6f7bd15/guardrails/validators.py#L21